### PR TITLE
Update screenshot

### PR DESCRIPTION
**Pull request by GerbrichFerdinands**
- update GUI screenshot selecting the cord-19 database. 
- add GitHub username @bmkramer to contributors.md

The sreenshot belongs in the readme of the asreview-covid19 repo, therefore this pr goes along with[ a pr in that repository.](https://github.com/asreview/asreview-covid19/pull/16)

### CODE REVIEW

**Comment by GerbrichFerdinands** [comment]
I've also updated the screenshots in the quicktour.

File: docs/source/quicktour.rst [hunk 1]
```diff
@@ -125,16 +132,19 @@ While you review the publications that the software presents you with,
...
-.. figure:: ../images/5_reviewing.png
+.. figure:: ../images/5.1_start_reviewing.png
    :alt:
 
-To view your progress, click the Statistics symbol in the upper right corner.
-The Statistics panel will open, showing information on the current
-project and labelling progress.
 
-.. figure:: ../images/statistics_1.png
+The Statistics panel shows information on the current project and labelling progress.
```
Comment (J535D165) [inline docs/source/quicktour.rst:139]: Labeling

File: docs/source/quicktour.rst [hunk 2]
```diff
@@ -125,16 +132,19 @@ While you review the publications that the software presents you with,
...
    :alt:
 
-To view your progress, click the Statistics symbol in the upper right corner.
-The Statistics panel will open, showing information on the current
-project and labelling progress.
 
-.. figure:: ../images/statistics_1.png
+The Statistics panel shows information on the current project and labelling progress. 
+The panel can be opened and closed during labeling as you like. 
+To hide your progress, click the Statistics symbol in the upper right corner.
```
Comment (J535D165) [inline docs/source/quicktour.rst:141]: This is the closing arrow, isn't it? Statistics symbol is for panel expansion.

**Comment by J535D165** [comment]
Step 5 in the master branch

localhost_3000_(iPad) (36)

File: docs/source/quicktour.rst (hunk 2 above)
Comment (GerbrichFerdinands) [reply docs/source/quicktour.rst:141]: Whoops I mixed up some sentences there. Thanks

**Comment by J535D165** [comment]
https://github.com/asreview/asreview/blob/master/images/asrewiew-plugin-cord19-dataset.png?raw=true

This link is also found in the COVID-19 plugin page.

**Comment by GerbrichFerdinands** [comment]
The link you mention above is a screenshot of an outdated version of the GUI. 

With covid-19 plugin page, do you mean the github repository? 
Anyqay, I made a PR to the COVID-19 plugin repo, replacing this link with a link to an updated version of the screenshot (see link to pr I mention on the top of this PR).

**Comment by J535D165** [comment]
This one: https://raw.githubusercontent.com/asreview/asreview/master/docs/source/covid-19.rst. The front page for AllenAI and Rijksoverheid.

**Comment by J535D165** [comment]
Thanks @GerbrichFerdinands.

**Pull request merged**